papi/handlers.go

```diff
@@ -463,6 +463,105 @@ func (h handler) GetMonthlyInfosByYear(c echo.Context) error {
 	return c.JSON(http.StatusOK, sumMI)
 }
 
+func (h handler) V2GetAggregateIndexes(c echo.Context) error {
+	jurisdicao := c.QueryParam("jurisdicao")
+	orgao := c.QueryParam("orgao")
+	agregado := c.QueryParam("agregado")
+
+	groupMap := map[string]string{
+		"justica-eleitoral":    "Eleitoral",
+		"ministerios-publicos": "Ministério",
+		"justica-estadual":     "Estadual",
+		"justica-do-trabalho":  "Trabalho",
+		"justica-federal":      "Federal",
+		"justica-militar":      "Militar",
+		"justica-superior":     "Superior",
+		"conselhos-de-justica": "Conselho",
+	}
+
+	var indexes map[string][]models.IndexInformation
+	var err error
+
+	// Verificamos se a jurisdição foi informada e se é válida
+	if groupName, ok := groupMap[jurisdicao]; ok {
+		indexes, err = h.client.Db.GetIndexInformation(groupName)
+		if err != nil {
+			return c.JSON(http.StatusBadRequest, fmt.Sprintf("Erro consultando os índices para o grupo %s.", groupName))
+		}
+		// Verificamos se o órgão foi informado
+	} else if orgao != "" {
+		indexes, err = h.client.Db.GetIndexInformation(orgao)
+		if err != nil {
+			return c.JSON(http.StatusBadRequest, fmt.Sprintf("Erro consultando os índices para o órgão %s.", orgao))
```

Comment:
Mesmo comentário acima com relação ao status de retorno.

Reply:
Baseado no retorno, você vai precisar checar se o órgão passado foi inválido e separar o que for badrequest e o que for um erro do backend (bd)